app/scripts/templates/global_header/global_header.html

```diff
@@ -1,4 +1,4 @@
-<div class="row-inner l-flex-horizontal">
+<div class="row-inner-padded l-flex-horizontal">
```

Comment:
This is semi unrelated, but I would prefer that we include `.l-flex-horizontal` in `.row-inner` rather than including it here. That way the CSS contains all the logic about layout rather than leaking those into the HTML. Let's discuss this more next week...